mods/content/plant_dissection/plants_fruit.dm

```diff
@@ -0,0 +1,113 @@
+/obj/item/food/fruit
```

Comment:
possibly able to be mixed up with `/obj/item/food/grown`, unsure what to do about that. `/obj/item/food/grown/fruit`? this seems to be like... totally unrelated to seeds and stuff though, so i don't know what to do about that.